docs/SPEC.md

```diff
@@ -350,8 +350,51 @@ The regular expression syntax is defined by [RE2](https://github.com/google/re2/
 
 ### Variables
 
-A variable holds a value.
-A variable can only hold values defined by its type.
+A variable represents a storage location for a single value.
+Variables are immutable.
+Once a variable is given a value, it holds that value for the remainder of its lifetime.
+
+### Options
+
+An option represents a storage location for any value of a specified type.
+Options are mutable.
+An option can hold different values during its lifetime.
+
+Below is a list of all options that are currently implemented in the Flux language:
```

Comment:
Since options can be arbitrarily created, I think we should word this as here are "some" built in options. This way we do not imply that this is the list of all options.